test/fakes/fake_just_audio.dart

```diff
@@ -137,8 +142,10 @@ class MockAudioPlayer implements AudioPlayerPlatform {
     }
     _audioSource = audioSource;
     _index = request.initialIndex ?? 0;
-    // Simulate loading time.
-    await Future<dynamic>.delayed(const Duration(milliseconds: 100));
+    if (binding.inTest) {
+      // Simulate loading time.
+      await Future<void>.delayed(const Duration(milliseconds: 100));
```

Comment:
Can you explain this change?

Reply:
See, my other response. I added the `binding.inTest` check because it occurred to me that outside of the test we don't need to delay.